packages/account-v2/src/pages/ConnectedApps/ConnectedAppsRevokeModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv-com/ui';
+import TrashIcon from '../../assets/connectedApps/ic-account-trash-can.svg';
+
+type TConnectedAppsRevokeModalProps = {
+    handleRevokeAccess: () => void;
+    handleToggleModal: (appId?: number | null) => void;
+    isModalOpen: boolean;
+};
+
+export const ConnectedAppsRevokeModal = ({
+    handleRevokeAccess,
+    handleToggleModal,
+    isModalOpen,
+}: TConnectedAppsRevokeModalProps) => (
+    <Modal className='p-24 md:w-[440px] sm:w-[328px] h-auto rounded-default' isOpen={isModalOpen}>
```

Comment:
Can u check if there is any console errors? While using Modal u need to set `Modal.setAppElement(ACCOUNT_MODAL_REF);`